test/IISIntegration.FunctionalTests/HelloWorldTest.cs

```diff
@@ -20,31 +19,23 @@ public HelloWorldTests(ITestOutputHelper output) : base(output)
         {
         }
 
-        [ConditionalTheory]
-        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
```

Comment:
These get disabled via KoreBuild: https://github.com/aspnet/IISIntegration/blob/dev/build/repo.props#L4. That said, this will cause the test runs to fail if you explicitly call `dotnet test` on this from a xplat machine, but that seems uncommon.